### PR TITLE
Adding Travis-Ci Support For Arm64

### PR DESCRIPTION
**Pull request by ossdev07**
Travis-CI has added support for ARM64. Added ARM64 job in Travis-CI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,13 +3,18 @@ cache: pip
...
+      - { python: '3.5', env: }
+      - { arch: arm64, python: '3.5', env: }
+      - { python: '3.6', env: }
+      - { arch: arm64, python: '3.6', env: }
+      - { python: '3.7', env: }
+      - { arch: arm64, python: '3.7', env: }
+      - { python: '3.8', env: }
+      - { arch: arm64, python: '3.8', env: }
+      - { python: 'pypy', env: }
+      - { python: 'pypy3', env: }
```
Comment (hugovk) [inline .travis.yml]: How about removing the redundant envs?

File: .travis.yml [hunk 2]
```diff
@@ -3,13 +3,18 @@ cache: pip
...
   include:
-    - python: 2.7
-    - python: 3.5
-    - python: 3.6
-    - python: 3.7
-    - python: 3.8
-    - python: pypy
-    - python: pypy3
+      - { python: '2.7', env: }
+      - { arch: arm64, python: '2.7', env: }
```
Comment (hugovk) [inline .travis.yml]: Put python first for readability

**Comment by hugovk** [comment]
What's the benefit on testing on ARM64? Is there something in this project which might have different results on different CPUs, and is it necessary to test all Python versions?

What about the other two CPUs? 

* https://blog.travis-ci.com/2019-11-12-multi-cpu-architecture-ibm-power-ibm-z

**Comment by rhenwood-arm** [comment]
@hugovk : I have been working with @ossdev07 on a down-stream project on AArch64 (ARM64) that has olefile as a dependency. olefile is a pure python project with a straightforward travis-ci build job and that doesn't have a binary release. In this case, there is very little benefit in testing on alternative architectures, and this PR can probably be held in reserve until it is needed.

**Comment by decalage2** [comment]
It looks like this PR is not really necessary, so I will close it for now. If you find a specific issue with ARM64 in the future, please reopen it.

**Pull request closed without merging**